config/deployer.sample.json

```
@@ -34,7 +34,7 @@
   "AdminEmail": "sysadmin@sample.mattermost.com",
   "AdminUsername": "sysadmin",
   "AdminPassword": "Sys@dmin-sample1",
-  "LoadTestDownloadURL": "https://github.com/mattermost/mattermost-load-test-ng/releases/download/v1.1.0/mattermost-load-test-ng-v1.1.0-linux-amd64.tar.gz",
+  "LoadTestDownloadURL": "https://github.com/mattermost/mattermost-load-test-ng/releases/download/1.2.0/mattermost-load-test-ng-1.2.0-linux-amd64.tar.gz",
```

Shall we keep existing notation by using: 
```suggestion
  "LoadTestDownloadURL": "https://github.com/mattermost/mattermost-load-test-ng/releases/download/v1.2.0/mattermost-load-test-ng-v1.2.0-linux-amd64.tar.gz",
```

---

Thanks for spotting this. This is actually an invalid link.
I've erroneously called the `make release` command with `NEXT_VER=1.2.0` instead of `NEXT_VER=v1.2.0`.
I'll fix this.